support-frontend/assets/pages/contributions-landing/contributionsLandingReducer.ts

```diff
@@ -65,9 +65,59 @@ export type StripeCardFormData = {
 	createPaymentMethod: ((clientSecret: string | null) => void) | null;
 	handle3DS: ((clientSecret: string) => Promise<Stripe3DSResult>) | null; // For single only
 };
+type Wallet = {
+	bind: (ID: string) => void;
+};
+type Consent = {
+	bind: (ID: string) => void;
+};
+export type BaseWalletConfig = {
```

Comment:
should we prefer `interface` over `type` in TS?

Reply:
I went with `type` as the the other types in `contributionsLandingReducer.ts` used `type`, but if the convention is to prefer interface I can switch these (and update the rest of the definitions in this module).